libraries/support/azure/management.rb

```diff
@@ -94,6 +94,14 @@ def security_center_policies
       )
     end
 
+    def storage_account(resource_group, name)
+      get(
+        url: link(location: 'Microsoft.Storage/storageAccounts',
+                  resource_group: resource_group) + name,
```

Comment:
Can we use ```"Microsoft.Storage/storageAccounts/#{name}"``` instead? I know there still some examples of concatenation but I think interpolation is best (https://stackoverflow.com/a/10076632)

Reply:
If were to try to interpolation here it would be something like:
```
url: "#{link(location: ''Microsoft.Storage/storageAccounts',
                  resource_group: resource_group)}#{name}"
```
Would it make more sense to change link to take in a name?
```
        url: link(location: 'Microsoft.Storage/storageAccounts',
                  resource_group: resource_group,
                  name: name),
```

Then we can just assume link always returns a fully crafted URL.

Reply:
Instead of using the `link` method defined in the file?

Reply:
I was proposing changing the the link method to handle a `name` key. 

```
    def link(location:, provider: true, resource_group: nil, name: nil)
      "/subscriptions/#{subscription_id}" \
      "#{"/resourceGroups/#{resource_group}" if resource_group}" \
      "#{'/providers' if provider}" \
      "/#{location}/" \
      "#{name if name}"
end
```
and
```

    def storage_account(resource_group, name)
      get(
        url: link(location: 'Microsoft.Storage/storageAccounts',
                  resource_group: resource_group,
                  name: name),
        api_version: '2017-06-01',
      )
end
```
```

Reply:
I'm not a fan of adding any more fields to the `link` method to be honest.
Current params, `subscription`, `resource_group` and `provider` are used in most API calls, but `name` would only be used in at most 50% (singular v. plural).

What is wrong with:
```
    def storage_account(resource_group, name)
      get(
        url: link(location: "Microsoft.Storage/storageAccounts/#{name}",
                  resource_group: resource_group),
        api_version: '2017-06-01',
      )
    end
```